tests/runtests.py

```diff
@@ -190,14 +190,24 @@ def __init__(self):
             'interactive/test_wx': ['wx'],
             }
 
-        # Other dependecies of some tests.
+        # Other dependecies of some tests. A value of None indicates that the dependency is satisfied; any other value is taken as as failed dependency and should be an string containing an error message.
         self.DEPENDENCIES = {
             'basic/test_onefile_ctypes': [depend.c_compiler()],
             # Support for unzipped eggs is not yet implemented.
             # http://www.pyinstaller.org/ticket/541
             'import/test_eggs1': ['Unzipped eggs not yet implemented.'],
+            'libraries/test_usb': [self._depend_usb()],
             }
 
+    # See if the usb package is supported on this platform.
+    def _depend_usb(self):
+        try:
+            import usb
+            # This will verify that the backend is present; if not, it will raise an exception.
+            usb.core.find(find_all = True)
+        except:
```

Comment:
What kind of exception is raises? Catch-all `except`-statements tend to hide other errors. So his should be restricted as far as possible.